src/lib/telldus-accessory.js

```diff
@@ -185,12 +221,23 @@ class TelldusDimmer extends TelldusSwitch {
    * @param  {object}             context
    */
   setDimLevel(value, callback, context) {
-    TDtool.dim(percentageToBits(value), this.id).then(out => {
-      return out.indexOf('Success') > -1 ? callback() : Promise.reject(out)
+    this.log('setDimLevel called')
+    this.dimlevel = value
+    if (this.dimlevel > 0){
```

Comment:
As far as I can se, the rest of this function can be skipped, and replaced with a call to setState:
```js
this.setState(value, callback, context)
```